oem/ibm/configurations/bios/enum_attrs.json

```diff
@@ -573,6 +573,21 @@
             "default_values": ["MTU1500"],
             "helpText": "Specifies the maximum frame size (maximum transmission unit, MTU). The value is in terms of bytes per packet size.",
             "displayName": "Maximum Frame Size"
+        },
+        {
+            "attribute_name": "pvm_linux_kvm_memory",
+            "possible_values": ["Automatic", "Custom"],
+            "default_values": ["Automatic"],
+            "helpText": "Controls whether the system or user specified percentage of available system memory will be reserved for the management of KVM guests.",
```

Comment:
Isn't it better to write the complete helptext that is mentioned in attribute doc as it enhances understanding

Reply:
no we dont need to , that was discussed in our meetings with GUI and host

Reply:
there is no need to explain definition of custom and automatic in help text